src/extensions/default/InlineColorEditor/InlineColorEditor.js

```diff
@@ -97,25 +88,24 @@ define(function (require, exports, module) {
      * @return {?{start:{line:number, ch:number}, end:{line:number, ch:number}}}
      */
     InlineColorEditor.prototype.getCurrentRange = function () {
-        var start, end;
+        var pos, start, end;
         
-        start = this._startBookmark.find();
+        pos = this._marker.find();
```

Comment:
I don't think it can ever be `null` - `cm.markText()` always returns a `TextMarker` instance, and we initialize it in the constructor and only `.clear()` it on close, so when could this ever be `null`?
Of course, `this._marker.find()` *can* be `null`.